lib/urldata.h

```diff
@@ -980,6 +983,9 @@ struct connectdata {
   void *seek_client;            /* pointer to pass to the seek() above */
 
   /*************** Request - specific items ************/
+#ifdef SECPKG_ATTR_ENDPOINT_BINDINGS
```

Comment:
Probably also needs `USE_WINDOWS_SSPI` like in the other conditions?

Reply:
Thanks for the feedback. Code updated!